src/jswrap_interactive.c

```diff
@@ -507,7 +508,10 @@ void _jswrap_interface_clearTimeoutOrInterval(JsVar *idVar, bool isTimeout) {
       jsvRemoveChild(timerArrayPtr, child);
       jsvUnLock2(child, timerArrayPtr);
     } else {
-      jsExceptionHere(JSET_ERROR, isTimeout ? "Unknown Timeout" : "Unknown Interval");
+      if (isTimeout)
+        jsExceptionHere(JSET_ERROR, "Unknown Timeout");
```

Comment:
I modified jsExcpetionHere to a macro that places the format string into flash, which requires the second argument to be a string constant. I looked into putting together a second macro that allows the format string to be any expression (like here), but then noticed that there are only 3-4 places in the codebase where it's not a constant string, so I decided it was cleaner overall to just modify these places rather than add the confusion of a second macro. I tried to keep the non-esp8266 code as straight-forward as possible while adding strings-in-flash support.